pages/editpage/[id].js

```diff
@@ -1,5 +1,5 @@
-import CreateRecipe from "../../components/Form";
+import EditRecipe from "../../components/Form/Edit";
 
-export default function CreatePage() {
-  return <CreateRecipe />;
+export default function EditPage() {
+  return <EditRecipe />;
```

Comment:
As I mentioned above, here you should do the fetch and define a submithandler and pass it as props to the ``<EditRecipe>``-component.